assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -529,7 +528,7 @@ che.infra.openshift.route.labels=NULL
 # plugins dependencies to a workspace
 #
 # Note these images are overridden by the Che Operator by default; changing the images here will not
-# have an effect if Che is installed via Operator.
+# Affect if Che is installed via Operator.
```

Comment:
`... changing the images here will not affect if Che is installed via Operator.` doesn't sound right to me.

Reply:
Maybe `will not affect the Che server if installed via Operator`?